proxy/envoy/testdata/envoy-ingress.json.golden

```diff
@@ -1,30 +1,18 @@
 {
   "listeners": [
     {
-      "address": "tcp://0.0.0.0:8080",
+      "address": "tcp://0.0.0.0:80",
       "filters": [
         {
           "type": "read",
           "name": "http_connection_manager",
           "config": {
             "codec_type": "auto",
             "stat_prefix": "http",
-            "route_config": {
-              "virtual_hosts": [
-                {
-                  "name": "foo.default.svc.cluster.local",
-                  "domains": [
-                    "foo.default.svc.cluster.local"
-                  ],
-                  "routes": [
-                    {
-                      "prefix": "/bar",
-                      "host_rewrite": "foo.default.svc.cluster.local:80",
-                      "cluster": "out.foo.default.svc.cluster.local|http"
-                    }
-                  ]
-                }
-              ]
+            "rds": {
+              "cluster": "rds",
+              "route_config_name": "80",
```

Comment:
This looks great! Now, where does this RDS gets its routes from? The manager ? If so, does that manager listen to ingress resource and generate the Envoy route config accordingly ? (Assuming that, it's still incomplete right, as we have no route rule support. But that's for another PR).